Clients/GUI/Pages/SignIn.razor

```diff
@@ -35,13 +32,15 @@
     bool isHidden = true;
     string errorMessage = "";
 
-    private void TrySignIn()
+    private async Task TrySignIn()
     {
-        var user = new UserEmailPassword(userInput.Email, userInput.Password);
         try
         {
-            // TODO: add logic after MassTransit flow iw merged
-            //SignInner.SignIn(user);
+            await LocalStorage.ClearAsync();
+            var hash = ShaHash.GetHash(userInput.Password);
+            var credentials = new UserEmailPassword(userInput.Email, hash);
+            var userToken = SignInner.SignIn(credentials);
+            LocalStorage.SetItemAsync(userToken.UserId.ToString(), userToken.Body);
```

Comment:
Why github isn' satisfied?

Reply:
> Why github isn' satisfied?

About what?

Reply:
Look at the files changed

Reply:
We don't really need  to wait until record is added to storage